pkgs/tools/misc/zf/default.nix

```diff
@@ -0,0 +1,36 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, zig
+}:
+stdenv.mkDerivation rec {
+  pname = "zf";
+  version = "0.5";
+
+  src = fetchFromGitHub {
+    owner = "natecraddock";
+    repo = pname;
+    rev = version;
+    sha256 = "sha256-df7fK3aYoMP4x0XWscYb9xrHl0tl8L9GcgCDQ6lTTnc=";
+  };
+
+  buildInputs = [ zig ];
+
+  buildPhase = ''
+    export HOME="$(mktemp -d)"
+    zig build -Drelease-fast=true
```

Comment:
```suggestion
    zig build -Drelease-fast=true -Drelease-safe -Dcpu=baseline
```

Reply:
I think it should have been `zig build -Drelease-fast -Dcpu=baseline`.  Release fast disables runtime safety checks for extra performance which seems to be a selling point of this tool.

Reply:
@McSinyx is correct. Zig release modes are mutually exclusive.

I recommend `-Drelease-fast` because it will crash on most errors either way, but in a fast release the fuzzy finder is much more useful. (I'm the author of zf, and I use it daily and haven't seen a crash in months anyway)

Reply:
I was curious and looked into this more: https://github.com/natecraddock/zf/issues/30

It looks like the performance difference between release safe and release fast is negligible. I'll let y'all decide which to use though.